src/SolidityLauncher.ts

```diff
@@ -423,6 +427,14 @@ export class SolidityLauncher {
     console.log = old;
     await this.api.onTestsComplete(this.config);
 
+    const mochaRunner: Runner = this.truffle.test.mochaRunner;
```

Comment:
What does this have to do with the generation of variable names bug?

Reply:
Well while trying to debug it was not printing that a testcase had failed.
So I added this. I think its good to keep it in here.